src/base/pegasus_const.h

```diff
@@ -68,4 +68,6 @@ extern const std::string ROCKSDB_BLOCK_CACHE_ENABLED;
 extern const std::string SPLIT_VALIDATE_PARTITION_HASH;
```

Comment:
Could you add some tests for this pull request?

Reply:
I will add some function tests for all read/write throttle tests in my next PR